package.json

```diff
@@ -35,6 +35,8 @@
     "shortid": "2.2.16"
   },
   "devDependencies": {
+    "@types/mocha": "^10.0.1",
+    "@types/node": "^20.5.9",
     "bun": "^0.8.1",
     "chai": "4.2.0",
```

Comment:
we probably don't need this one either as bun has jest expect syntax built in.

Reply:
Feel free to remove :)